packages/frontend/src/app/breadcrumbs/breadcrumbs.component.css

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#main-container {
+    display: flex;
+    justify-content: center;
+}
+#crumbs-container {
+    width: 100%;
+    max-width: 800px;
+    margin-top: 5px;
+}
+
+#home-icon {
+    font-size: 1.2em;
+    vertical-align: middle;
+    display: inline-flex;
+    align-items: center;
+    padding-bottom: 3px;
+    width: 18px;
```

Comment:
To @chingor13's drive by comment the other day, I would be curious if there are more off the shelf styles we can use.

Mind making this a topic of conversation when you have a 1:1 with Martin?

Reply:
Sure will !